src/App.js

```diff
@@ -1,23 +1,22 @@
-import logo from './logo.svg';
-import './App.css';
+import { Route, Routes } from "react-router-dom";
+import "./App.css";
+import BeerDetails from "./components/BeerDetails";
+import Home from "./components/Home";
+import "bootstrap/dist/css/bootstrap.min.css";
+import RandomBeer from "./components/RandomBeer";
+import NewBeer from "./components/NewBeer";
+import AllBeers from "./components/AllBeers";
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Routes>
+        <Route path="/" element={<Home />}></Route>
+        <Route path="/beers" element={<AllBeers />}></Route>
+        <Route path="/beers/:id" element={<BeerDetails />} />
+        <Route path="/random" element={<RandomBeer />} />
```

Comment:
That route needs to match the `Link` that you have in the `Home` component or the other way around  

```suggestion
        <Route path="/beers/random" element={<RandomBeer />} />
```